salesforce/query.py

```diff
@@ -0,0 +1,34 @@
+import json
+import os
+
+
+from helpers.auth import client
+
+
+def main():
+    query = os.getenv("QUERY")
+    if query is None:
+        print("No query provided")
+        exit(1)
+
+    sf = client()
+    res = sf.query(query)
+    if res.get("totalSize") > 0:
+        print_res(res)
+        next_url = res.get("nextRecordsUrl")
+        while next_url:
+            res = sf.query_more(next_url, True)
+            next_url = res.get("nextRecordsUrl")
+            print_res(res)
+    else:
+        print("No records found")
+        exit(1)
```

Comment:
nit: I don't think this makes a difference to GPTScript, but this seems like an exit(0) situation to me

```suggestion
        exit(0)
```